cfgov/unprocessed/apps/owning-a-home/js/explore-rates/data-loader.js

```diff
@@ -29,9 +29,11 @@ function getData( fieldToFetch ) {
     cancelFunc = newCancelFunc;
   } );
 
-  const params = Object.assign(
-    { decache: decache, cancelToken: _cancelToken }, fieldToFetch
-  );
+  const params = { decache: decache, cancelToken: _cancelToken };
+  // This could use Object.assign, but it's not supported in IE11.
```

Comment:
Just use the assign utility

Reply:
Worth it? The object is so tiny.

Reply:
Leaving without an external dep for now to fix it.